Formula/gawk.rb

```diff
@@ -49,11 +51,24 @@ def install
     end
     system "make", "install"
 
+    (bin/"awk").unlink if OS.mac?
     (libexec/"gnubin").install_symlink bin/"gawk" => "awk"
     (libexec/"gnuman/man1").install_symlink man1/"gawk.1" => "awk.1"
     libexec.install_symlink "gnuman" => "man"
   end
 
+  def caveats
+    on_macos do
+      <<~EOS
+        GNU "awk" has been installed as "gawk".
+        If you need to use it as "awk", you can add a "gnubin" directory
+        to your PATH from your ~/.bashrc and/or ~/.zshrc like:
+
+            PATH="#{opt_libexec}/gnubin:$PATH"
```

Comment:
Modified from `make`'s caveats, which reads:

https://github.com/Homebrew/homebrew-core/blob/4f0a57b6b1da479dd19c8570824aab5f251ad704/Formula/make.rb#L40-L50

Understand that this was written when Bash was still the default shell on macOS. And there are 16 occurrences of "to your PATH from your bashrc". Should they be changed also?

They are: `couchdb-lucene`, `ed`, `findutils`, `gnu-indent`, `gnu-sed`, `gnu-tar`, `gnu-time`, `gnu-units`, `gnu-which`, `grep`, `inetutils`, `libtool`, `make`, `man-db`, `uutils-coreutils`, `uutils-findutils`.